File: Sources/SimERP/ClientApp/src/app/lists/exchangerate/model/exchangerate.ts
export class ExchangeRate {
  ExchangeRateId = 0;
  CurrencyId = -1;
  ExchangeRating: number;
  ExchangeDate: Date;
  Notes: string;

  // custom properties
  CurrencyName: string;
  ExchangeDateNonOffset: Date;
}

File: Sources/SimERP.Data/DBEntities/ExchangeRate.cs
using System;
using System.Collections.Generic;

namespace SimERP.Data.DBEntities
{
    public partial class ExchangeRate
    {
        public int ExchangeRateId { get; set; }
        public string CurrencyId { get; set; }
        public decimal ExchangeRating { get; set; }
        public DateTimeOffset? ExchangeDate { get; set; }
        public string Notes { get; set; }
    }
}
